prow/plugins/approve/approve.go

```diff
@@ -312,7 +312,8 @@ func handlePullRequest(log *logrus.Entry, ghc githubClient, oc ownersClient, git
 
 // Returns associated issue, or 0 if it can't find any.
 // This is really simple, and could be improved later.
-func findAssociatedIssue(body string) int {
+func findAssociatedIssue(body, org string) int {
+	associatedIssueRegex := regexp.MustCompile(fmt.Sprintf(associatedIssueRegexFormat, org))
```

Comment:
It isn't safe to use `MustCompile` here since this could panic. `MustCompile` is typically only used at init time, please use `Compile` here instead and handle the error.

Reply:
Done.